app/src/services/email.ts

```diff
@@ -1,9 +1,20 @@
 import axios from 'axios';
 import config from 'config';
+import prisma from '../db/dataConnection';
```

Comment:
1st party import should be separate from 3rd party

Reply:
Moved import down to be separate.